<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WT_ProjectWidget</class>
 <widget class="QWidget" name="WT_ProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_25">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="horizontalSpacing">
         <number>8</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="4">
         <widget class="MonitoStateW" name="MonitState3" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>182</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="MonitoStateW" name="MonitState1" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>182</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="MonitoStateW" name="MonitState4" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>182</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="MonitoStateW" name="MonitState2" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>182</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <spacer name="verticalSpacer_28">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_29">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <spacer name="verticalSpacer_30">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_31">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <spacer name="verticalSpacer_26">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_27">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_18">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MonitoStateW</class>
   <extends>QWidget</extends>
   <header>monitostatew.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
